subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -361,6 +392,12 @@ static int job_update_accepted(struct mqtt_client *const client,
 		reset_library();
 	}
 		break;
+	case AWS_JOBS_FAILED: {
```

Comment:
Are the curly braces needed in this case? I think thats only needed when declaring variables inside the switch case.